_posts/2024-11-20-cockpit-329.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 329
+author: mvo
+date: '2024-11-20'
+tags: cockpit, files
+slug: cockpit-329
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit 329 and cockpit-files 12:
+
+
+## Shell: Extra warnings when connecting to remote hosts
+
+Connecting to multiple hosts in a single Cockpit session allows all
+these hosts to access each other freely. Eventually Cockpit will not
+allow multiple connections, but in the mean time, we want to educate
+people better.
+
+![screenshot of extra warnings when connecting to remote hosts]({{ site.baseurl }}/images/329-shell-extra-warnings-when-connecting-to-remote-hosts.png)
+
+This warning can be disabled by including the following in `/etc/cockpit/cockpit.conf`:
+
+```
+[Session]
+WarnBeforeConnecting=false
+```
+
+## Files: File creation support
+
+After creation the dialog changes into editor mode so you can keep
+editing. When you have "administrative access" the owner of the file
+is pre-selected based on the owner of the directory.
+
+![screenshot of file creation support]({{ site.baseurl }}/images/329-cockpit-files-file-creation-support.png)
+
+## Try it out
+
+Cockpit 329, cockpit-machines 324, and cockpit-files 12 are available now:
```

Comment:
Let's drop machines: It's not in the initial paragraph nor in the list below.

Reply:
Ouch, yes. Done.